docs/static/ls-ls-config.asciidoc

```diff
@@ -0,0 +1,111 @@
+[[ls-to-ls]]
+=== Configuring Logstash-to-Logstash Communication
+
+You can set up communication between two Logstash machines by connecting the Lumberjack output to the Beats input. You may need this configuration if the data path crosses network or firewall boundaries, for example. If you don't have a compelling need for Logstash-to-Logstash communication, then don't implement it.
+
+[[ls-to-ls-overview]]
+==== Configuration overview
+
+Use the Lumberjack protocol to connect two Logstash machines.
+
+. Generate a trusted SSL certificate (required by the lumberjack protocol).
+. Save the SSL certificate on the upstream Logstash machine.
+. Save the SSL certificate and key on the downstream Logstash machine.
+. Set the downstream Logstash machine to listen for incoming Lumberjack connections through the Beats input.
+. Set the upstream Logstash machine to use the Lumberjack output to send data.
+. Test it.
+
+[[generate-self-signed-cert]]
+===== Generate a self-signed SSL certificate and key
+
+You can use the `openssl req` command to generate a self-signed certificate and key. The `openssl req` command is available on any operating system that comes with SSL.
```

Comment:
Not really true that the command is available on any operating system with SSL. We can replace the last sentence with: "You may need to install the openssl command line program".